addons/udes_stock/tests/test_picking_batch.py

```diff
@@ -1705,3 +1705,213 @@ def test04_rereservation_for_same_batch(self):
 
         batch.reserve_pallet('UDES11111')
         batch.reserve_pallet('UDES11111')
+
+
+class TestMultipleOrders(common.BaseUDES):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.pack_4apples_info = [{'product': cls.apple,
+                                  'qty': 4}]
+        cls.pack_4bananas_info = [{'product': cls.banana,
+                                  'qty': 4}]
+
+        cls.picking_type_pick.u_reserve_pallet_per_picking = True
+
+    def setUp(self):
+        super().setUp()
+        self.create_quant(self.apple.id,
```

Comment:
If we are creating the same quants for every test, can we just add it to setUpClass?